main.go

```diff
@@ -293,6 +286,8 @@ and reports back results to the queue.
                                             [default: taskcluster-proxy]
           taskclusterProxyPort              Port number for taskcluster-proxy HTTP requests.
                                             [default: 80]
+          taskclusterRootURL                Root URL of taskcluster cluster that this worker
+                                            should interact with. [default: https://taskcluster.net]
```

Comment:
I think a default is OK here temporarily, but once everything that launches workers is specifying a rootUrl, let's plan to remove the default.